src/CalendarInput/index.js

```diff
@@ -34,6 +34,7 @@ const SINGLE_SELECTION = 'single'
 const isPeriodSelection = equals(PERIOD_SELECTION)
 
 /**
+ * @deprecated
  * This component is a simplified version of the date selector only with the basic
  * and the calendar inputs.
  * The input component designed to receive one or two dates, have a date format mask
```

Comment:
@vagnervst add at the end of component description.

```suggestion
 * The input component designed to receive one or two dates, have a date format mask
 *
 * @deprecated CalendarInput component is being deprecated, use DateInput with showCalendar prop as "false" instead.
```

Reply:
Done!